x/authz/spec/03_services.md

```diff
@@ -0,0 +1,46 @@
+<!--
+order: 3
+-->
+
+# Tx Msg Service
+
+In this section we describe the Protobuf Msg service for the authz module.
```

Comment:
not sure about it. `x/authz` emphasis package name, "authz module" is a plain English. Do you have a preference?

Reply:
That makes sense. I do not have a strong preference. This would be a good example of something to explain in a styleguide for writing documentation/specifications (maybe added to SPEC-SPEC).